tsdef/twilio-video-tests.ts

```diff
@@ -103,3 +103,38 @@ function localDataTrackPublication(publication: Video.LocalDataTrackPublication)
   const publicationInfo = { kind, track };
   return publicationInfo;
 }
+
+function RemoteParticipant(RemoteParticipant: Video.RemoteParticipant) {
+  const chatLog = document.getElementById('someElementChat');
+  const RemoteData = RemoteParticipant.dataTracks;
+  const RemoteAudio = RemoteParticipant.audioTracks;
+  const sid = RemoteParticipant.sid;
+
+  const testDataTracks = RemoteParticipant.dataTracks;
+
+  RemoteParticipant.on('disconnected', remoteParticipant => {
+    return `${remoteParticipant} Has Disconnected`;
+  });
+  RemoteParticipant.on('trackMessage', msg => {
+    const textElement = document.createElement('p');
+    if (chatLog) {
+      textElement.innerText = `${msg}`;
+      chatLog.appendChild(textElement);
+    }
+  });
+}
+
+function LocalParticipant(LocalParticipant: Video.LocalParticipant) {
+  const LocalAudioTrack = LocalParticipant.audioTracks;
+  const LocalDataTracks = LocalParticipant.dataTracks;
+  const LocalVideoTrack = LocalParticipant.videoTracks;
+  const LocalTracks = LocalParticipant.tracks;
+  const signalingRegion = LocalParticipant.signalingRegion;
```

Comment:
Looks like these five variables (`LocalAudioTrack`, `LocalDataTrack`, `LocalVideoTrack`, `LocalTrack`, and `signalingRegion`) are assigned values, but they aren't used any where. What's the intent with these variables?

Reply:
Mostly just checking that they exist on the class! Do you think I should remove them? I included them just to make sure!